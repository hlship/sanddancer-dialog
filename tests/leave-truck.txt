leave
enter truck
leave truck
enter truck
get out
