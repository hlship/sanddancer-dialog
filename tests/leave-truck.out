

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> leave
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

The air is frigid, and turns your breath into clouds.

> enter truck
You get into the pickup truck.

> leave truck
You get out of the pickup truck.

> enter truck
You get into the pickup truck.

> get out
You get out of the pickup truck.

> 
